frontend/src/screens/wallet/Send.tsx

```diff
@@ -16,98 +10,123 @@ import {
   AlertTitle,
 } from "src/components/ui/alert.tsx";
 import { Button } from "src/components/ui/button";
-import {
-  Card,
-  CardContent,
-  CardFooter,
-  CardHeader,
-  CardTitle,
-} from "src/components/ui/card";
 import { Input } from "src/components/ui/input";
 import { Label } from "src/components/ui/label";
-import { LoadingButton } from "src/components/ui/loading-button";
 import { useToast } from "src/components/ui/use-toast";
+import BalanceCard from "src/components/wallet/BalanceCard";
+import ConfirmPayment from "src/components/wallet/ConfirmPayment";
+import LnurlPay from "src/components/wallet/LnurlPay";
 import { useBalances } from "src/hooks/useBalances";
 import { useChannels } from "src/hooks/useChannels";
 
 import { useInfo } from "src/hooks/useInfo";
-import { copyToClipboard } from "src/lib/clipboard";
-import { PayInvoiceResponse } from "src/types";
-import { request } from "src/utils/request";
+
+// email regex: https://emailregex.com/
+// modified to allow _ in subdomains
+const LIGHTNING_ADDRESS_REGEX =
+  /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-_0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+const BOLT11_REGEX = /^(lnbc|lntbs)([0-9a-z]+)$/i;
```

Comment:
where did you get this from? does this support standard testnet too or just mutinynet?

Reply:
I don't think we need this actually - we can just wrap creating the invoice in a try/catch and if it fails then we know it's not a valid invoice. Then this is handled by lightning-tools.